Integrations/DuoAdminApi/DuoAdminApi.py

```diff
@@ -70,7 +70,7 @@ def get_entry_for_object(title, obj, contents, context, headers=None, human_read
         }
 
     if headers:
-        if isinstance(headers, basestring):
+        if isinstance(headers, (str, unicode)):
```

Comment:
are you sure it's working?

Reply:
from the Python software foundation:

> This abstract type is the superclass for str and unicode. It cannot be called or instantiated, but it can be used to test whether an object is an instance of str or unicode. isinstance(obj, basestring) is equivalent to isinstance(obj, (str, unicode)).